### PR TITLE
Add new tool riboWaltz

### PR DESCRIPTION
**Pull request by pavanvidem**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by pavanvidem** [comment]
Solves https://github.com/galaxyproject/tools-iuc/issues/4170

File: tools/ribowaltz/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+categories:
```
Comment (bgruening) [inline tools/ribowaltz/.shed.yml:1]: since those are 2 tools, this needs to be suite defintion

Comment (pavanvidem) [reply tools/ribowaltz/.shed.yml:1]: Both of them are based on functions from the same R package. It should fit to both.

Comment (bgruening) [reply tools/ribowaltz/.shed.yml:1]: Yes, but we only want to have one tool per toolshed repo

**Comment by bgruening** [comment]
Its green :)

**Comment by bgruening** [comment]
> galaxy.tool_util.verify.interactor.JobOutputsError: Output collection 'out_plots': identifier 'read_ends_heatmap' found out of order, expected order of {3: 'metaplots', 2: 'psites_per_region', 1: 'read_ends_heatmap', 0: 'read_lengths', 4: 'trinucleotide_periodicity'} for the tool generated collection elements ['metaplots', 'psites_per_region', 'read_ends_heatmap', 'read_lengths', 'trinucleotide_periodicity']

That is the recent error

**Comment by pavanvidem** [comment]
that's strange!. From the error report it seems the collection elements are sorted lexicographically but the order in the test is not. I name sorted the elements in the test. I don't know  if that makes sense.

**Review by bgruening — APPROVED** [review]

**Comment by pavanvidem** [comment]
gracias!!

**Pull request merged**